rllib/utils/nested_dict.py

```diff
@@ -121,10 +121,10 @@ def __init__(
         x = x or {}
         if isinstance(x, NestedDict):
             self._data = x._data
-        elif isinstance(x, Mapping):
```

Comment:
This was a learning for me :)

Reply:
Can you elaborate? I'm curious.

Reply:
`isinstance(x, Mapping)` does not say check if x has the behavior as a Mapping type. It fails for example in tf.DictWrapper data types. but `isinstance(x, abc.Mapping)` fixes it. Same thing for `Iterable` vs. `abc.Iterable`

Reply:
Many thanks :)